src/test/unit_tests/test_bucketspace_fs.js

```diff
@@ -954,6 +955,65 @@ mocha.describe('bucketspace_fs', function() {
             assert.deepEqual(output_lifecycle, []);
         });
     });
+
+    mocha.describe('bucket policy ownership validations', function() {
+        mocha.before(async function() {
+            const param = { name: test_bucket3 };
+            await bucketspace_fs.create_bucket(param, dummy_object_sdk);
+        });
+        mocha.after(async function() {
+            await fs_utils.folder_delete(`${new_buckets_path}/${test_bucket}`);
+            const file_path = get_config_file_path(CONFIG_SUBDIRS.BUCKETS, test_bucket3);
+            await fs_utils.file_delete(file_path);
+        });
+        mocha.it('get bucket policy not allowed non owner user', async function() {
+            const policy = generate_s3_policy(account_user1.name, test_bucket3, ['s3:*']).policy;
+            const param = { name: test_bucket3, policy: policy };
+            await bucketspace_fs.put_bucket_policy(param, dummy_object_sdk);
+            const res = await bucketspace_fs.get_bucket_policy(param, dummy_object_sdk);
+            assert_bucket_policies(policy, res.policy);
```

Comment:
I think you can move these lines to the `mocha.before`, they are repeated in all the tests (`it`) under the `describe`.